spec/venue_spec.rb

```diff
@@ -44,12 +39,59 @@
 
   describe '#yell_at_patrons' do
     it 'returns a list of uppercased names' do
-      skip
       venue = Venue.new('Bluebird', 4)
       venue.add_patron('Mike')
       venue.add_patron('Megan')
       venue.add_patron('Bob')
       expect(venue.yell_at_patrons).to eq ['MIKE', 'MEGAN', 'BOB']
     end
   end
+
+  # Iteration 3 Test
+  describe '#over_capacity?' do
```

Comment:
These three tests are great. You can also combine them into a single test and test both the `true` and `false`. It might look like this:
```
describe "#over capcity" do
    it "returns true if venue has more patrons vs capacity" do
      venue = Venue.new('Bluebird', 4)
      venue.add_patron('Mike')
      venue.add_patron('Megan')
      venue.add_patron('Bob')

      expect(venue.over_capacity?).to eq(false)

      venue.add_patron('James')
      venue.add_patron('Cat')

      expect(venue.over_capacity?).to eq(true)
  end
```